roles/memcached/defaults/main.yml

```diff
@@ -19,3 +19,5 @@ memcached_helm_chart_ref: /usr/local/src/memcached
 memcached_helm_release_namespace: openstack
 memcached_helm_kubeconfig: "{{ kubeconfig_path | default('/etc/kubernetes/admin.conf') }}"
 memcached_helm_values: {}
+
+memcached_orphan_delete: "{{ atmosphere_orphan_delete }}"
```

Comment:
This is not a good approach IMHO, because with CI/CD in place, it's going to be impossibel for users to know when to use what, it should auto-detect and automatically enable as needed.